docs/features/techdocs/configuration.md

```diff
@@ -106,6 +106,11 @@ techdocs:
       # This allows providers like LocalStack, Minio and Wasabi (and possibly others) to be used to host tech docs.
       s3ForcePathStyle: false
 
+      # (Optional) AWS Server Side Encryption
+      # If not set, encrypted buckets will fail to publish.
+      # https://docs.aws.amazon.com/AmazonS3/latest/userguide/specifying-s3-encryption.html
+      sse: 'aws:kms' # or AES256
```

Comment:
Thanks for getting this documented!  Would you mind also adding this type to the [plugin-techdocs-backend config schema](https://github.com/backstage/backstage/blob/master/plugins/techdocs-backend/config.d.ts#L73)?  Specifying type there as `'aws:kms' | 'AES256'` would also be nice.